pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnMetadata.java

```diff
@@ -32,15 +33,20 @@
     private String nameWithCase;
     private String[] fieldNames;
     private Integer[] positionIndices;
+    private PulsarColumnHandle.HandleKeyValueType handleKeyValueType;
+    public final static String KEY_SCHEMA_COLUMN_PREFIX = "key.";
+    public final static String VALUE_SCHEMA_COLUMN_PREFIX = "value.";
```

Comment:
I don't think we need to add `value.` for value. We should keep the behavior consistent between messages with keys and without keys.

Reply:
Yes, it’s better! 👍

Reply:
Maybe we could use ```__key``` as the prefix of the key schema field name.

Reply:
Yes. +1